memgpt/cli/cli_config.py

```diff
@@ -26,6 +27,14 @@ def configure():
 
     MemGPTConfig.create_config_dir()
 
+    # this is to initialize them, so we do not get linter warnings
+    openai_key = ""
```

Comment:
Can we set this to `None` instead of empty strings, and also place them next to where the value is set? (e.g. like `default_agent` at line 114)

Reply:
I would use None if the empty string has some (different) meaningful value. But you can use None too I guess.

Reply:
The config parsing code currently assumes empty values are None, so we should set to that.

Reply:
That may be okay!

NOTE: I guess I do too much Go code where this could lead into problems if you also use ENV and the "function everybody uses to read them", which actually returns "" if the variable is not set, what classifies as a design error for me. It makes you aware that "" is not None and None is not "" :)

Reply:
>  and also place them next to where the value is set? 

I am not sure about that. The value should be set to anything before a potential first use. As linters do static analysis, it may be that it "technically" can't be used uninitialized but that can change with any refactoring and "I do not allow warnings in our code". But this is not "our" code of course :)